frontend/src/metabase/parameters/components/FilterApplyButton/FilterApplyButton.styled.ts

```diff
@@ -1,9 +1,16 @@
 import styled from "@emotion/styled";
 
-import { space } from "metabase/styled-components/theme";
+import { breakpointMaxSmall, space } from "metabase/styled-components/theme";
 import Button from "metabase/core/components/Button/Button";
 
 export const ApplyButton = styled(Button)`
   margin-left: auto;
   margin-top: ${space(1)};
+
+  ${breakpointMaxSmall} {
+    width: 100%;
+    /* make button same height as fields on small screens */
+    padding-top: 11px;
+    padding-bottom: 11px;
```

Comment:
This is what I asked @kdoh at the offsite. By using our new component library based on Mantine, we should not have to specify one-off dimensions like this 99% of the time in the future.